### ISSUE TITLE
Centered labels in horizontal bars

### ISSUE THREAD

**Issue by TheFizz** [ScottPlot/ScottPlot#3533]
**ScottPlot Version**: 5.0.22

Hey. Thank you very much for ScottPlot, very convenient and intuitive.

However, I am having a hard time aligning the labels inside horizontally stacked bars.
I am using a WinForms library (if this changes anything)

I can use offsetY like this
```
var barPlot = PlotView.Plot.Add.Bars(bars);
barPlot.Horizontal = true;
barPlot.ValueLabelStyle.OffsetY = 30;
```
To produce result like this
![image](https://github.com/ScottPlot/ScottPlot/assets/37963412/3252c510-bd49-4353-8e7c-37b82c0f82fb)

but the moment I change the number of bars it breaks =/
![image](https://github.com/ScottPlot/ScottPlot/assets/37963412/2a594419-4b93-464a-8053-8508511550ba)

This is not too surprising (given that the bar width had changed). On this note I have a couple of questions I didn't find answers to

1) Is it possible to set bars to constant width (so I can use the same offset)
2) Is it possible to read the bar width (and adjust offset accordingly)
3) Maybe there is an alignment option I missed? (I know there is `barPlot.ValueLabelStyle.Alignment = Alignment.MiddleCenter;` but it works weird)

This is without offset with `MiddleCenter` alignment
![image](https://github.com/ScottPlot/ScottPlot/assets/37963412/61521a23-9291-4f2e-8fbc-30442cdb44a4)

I appreciate your work
Thanks in advance
-Sev

**Comment by swharden** [comment]
Took a while to get to this one 😅 

I discovered that bar plots have a `CenterLabel` flag that does this, but it's presently not working as expected for horizontal bars. I'll fix this now 👍

**Comment by swharden** [comment]
Resolved in #4431 

```cs
ScottPlot.Palettes.Category10 palette = new();

// create 5 groups of stacked bars
for (int i = 0; i < 5; i++)
{
    // create 3 individual bars with stacking
    List<ScottPlot.Bar> bars = [];
    double valueBase = 0;
    for (int j = 0; j < 3; j++)
    {
        double barSize = Generate.RandomInteger(10, 20);
        ScottPlot.Bar bar1 = new()
        {
            FillColor = palette.GetColor(j),
            Position = i,
            ValueBase = valueBase,
            Value = valueBase + barSize,
            Label = $"{barSize}",
            CenterLabel = true,
        };
        bar1.LineStyle.AntiAlias = false;

        bars.Add(bar1);
        valueBase += barSize;
    }

    // plot the stacked bars
    var barPlot = myPlot.Add.Bars(bars);
    barPlot.Horizontal = true;
}

// style the plot so the bars start on the left edge
myPlot.Axes.Margins(left: 0);

// add custom group labels
double[] tickPositions = Generate.Consecutive(5);
string[] tickLabels = Enumerable.Range(1, 5).Select(x => $"Worker #{x}").ToArray();
myPlot.Axes.Left.SetTicks(tickPositions, tickLabels);
```

![image](https://github.com/user-attachments/assets/4f60a163-4bae-4272-ae88-f8a4689ff9f8)
